src/main/kotlin/me/moeszyslak/polly/data/MacroStore.kt

```diff
@@ -1,20 +1,64 @@
 package me.moeszyslak.polly.data
 
+import com.gitlab.kordlib.core.entity.channel.TextChannel
 import me.jakejmattson.discordkt.api.dsl.Data
 
 data class MacroStore(
-        val macros: MutableMap<GuildId, MutableMap<String, Macro>> = mutableMapOf()): Data("config/macros.json", killIfGenerated = false) {
+        val macros: MutableMap<GuildId, MutableMap<String, Macro>> = mutableMapOf()) : Data("config/macros.json", killIfGenerated = false) {
+    @Transient
+    val aliases: MutableMap<GuildId, MutableMap<String, Macro>> = mutableMapOf()
```

Comment:
I was thinking `MutableMap<String, String>` instead of `MutableMap<String, Macro>` but this is also fine.